test/astar.cc

```diff
@@ -40,14 +40,22 @@ namespace bpt = boost::property_tree;
 
 using namespace valhalla;
 namespace vm = valhalla::midgard;
-namespace vb = valhalla::baldr;
```

Comment:
Is adding a `using namespace valhalla::baldr;` in addition to the `vb` namespace definition what you'd expect?
This would minimize the code changes in this file?

Reply:
I think it's usually either/or so far, but having both is fair enough IMO, you choose:)